osrf/vrx
# Adding 2022/phase2 worlds and extra models

This pull request adds all the world files used during the `2022/phase2` event and some of the extra models required.

How to test it?

You can launch the simulation with:

```
roslaunch vrx_gazebo vrx.launch verbose:=true world:=/home/caguero/vrx2022_phase2_ws/src/vrx/vrx_gazebo/worlds/2022_phase2/gymkhana0.world
```

Feel free to launch and explore a few worlds to verify that the new models load (specially in the gymkhana tasks).
